rfcs/59-from_reflect_ergonomics.md

```diff
@@ -0,0 +1,284 @@
+# Feature Name: `from_reflect_ergonomics`
+
+## Summary
+
+Derive `FromReflect` by default when deriving `Reflect`, add `ReflectFromReflect` type data, and make `ReflectDeserializer` easier to use.
+
+## Terminology
```

Comment:
I feel like this might be clearer as a single section, comparing and constrasting at each step. Perhaps a table even?

Reply:
I tried the table, but it wasn't helping that much. Hopefully this is closer to what you wanted.